docs/platforms/dart/integrations/http-integration.mdx

```diff
@@ -4,6 +4,37 @@ description: "Learn more about the Sentry HTTP integration for the Dart SDK."
 sidebar_order: 2
 ---
 
+## Using the `runWithClient` function
+
+With [`runWithClient`](https://pub.dev/documentation/http/latest/http/runWithClient.html) you can define a global `SentryHttpClient` and run it in its own [`Zone`](https://api.dart.dev/stable/3.4.4/dart-async/Zone-class.html).
```

Comment:
Since you added the code snippet as a tab, now I don't think this runWithClient section is necessary anymore. Maybe you could add a short introduction/summary of the two alternatives the user could do to use the http integration. One is using the SentryHttpClient directly and the other is using the runWithClient function. Or something in that direction

Reply:
I rephrased this section. Please have a look at my latest commit.